assets/css/tailwind.css

```diff
@@ -3,8 +3,79 @@
 @tailwind utilities;
 
 @layer base {
-  h1 { @apply text-3xl }
-  h2 { @apply text-2xl }
-  h3 { @apply text-xl }
-  h4 { @apply text-lg font-semibold }
+  h1 { @apply text-3xl; }
+  h2 { @apply text-2xl; }
+  h3 { @apply text-xl; }
+  h4 { @apply text-lg font-semibold; }
+
+  :root {
+    /* These colors are using a syntax special to tailwind. Uncomment hex value for reference */
+    /* https://tailwindcss.com/docs/customizing-colors#using-css-variables */
+
+    /* general branding colors */
+    --color-primary: /* #ED6C5A */ 237 108 90;
+    --color-secondary: /* #245A7D */ 36 90 125;
+    --color-tertiary: /* #245A7D */ 36 90 125;
+    --color-primary-inverted: /* #ffffff */ 255 255 255;
+    --color-primary-hover: /* #d66051 */ 214 96 81;
+    --color-secondary-inverted: /* #ffffff */ 255 255 255;
+    --color-secondary-hover: /* #d66051 */ 214 96 81;
+    /* element specific colors */
+    --color-text-base: /* #252525 */ 37 37 37;
+    --color-text-muted: /* #999999 */ 153 153 153;
+    --color-text-inverted: /* #000000 */ 255 255 255;
+    --color-bg-primary: /* #fcfcfc */ 252 252 252;
+    --color-bg-secondary: /* #fff1ef */ 255 241 239;
+  }
+
+  .theme-violet {
```

Comment:
added support for multiple themes! 
I know we don't need this right now, but I had some fun learning about it.


Surprisingly, I'd like to keep them in here because they've been effective for css styling when using inverted colors. (especially the neon theme ⚡ )

If you want to see them, just add the class to the `app.vue` parent `div`